webpack/assets/javascripts/react_app/components/Settings/__tests__/SettingsActions.test.js

```diff
@@ -0,0 +1,41 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import Immutable from 'seamless-immutable';
+import { mock as mockApi } from '../../../mockRequests';
```

Comment:
I would like to suggest a different approach which I see as better, instead of using the `axios` mock, use `jest` mocking to mock the API itself.

See: https://github.com/theforeman/foreman/blob/c18d67464d81943b77feeb0e24ec649574860ab7/webpack/assets/javascripts/react_app/components/BreadcrumbBar/__tests__/BreadcrumbBarActions.test.js

Reply:
Changed the tests, thanks!